src/views/Dashboard.tsx

```diff
@@ -1,12 +1,39 @@
-import React from 'react';
+import React, { useEffect, useRef } from 'react';
 import { Container } from '../components/Container/Container';
+import { LeadsList } from 'components/LeadsList/LeadsList';
+import { fetchUserLeads } from 'data/leads/slice';
+import { useAppDispatch } from 'store/hooks';
 
 import styles from './Dashboard.module.scss';
 
 export const Dashboard = () => {
+   const dispatch = useAppDispatch();
+   const initialized = useRef(false);
+
+   useEffect(() => {
+      if (!initialized.current) {
+         initialized.current = true;
```

Comment:
**question:** Why do you need it for?

Reply:
Becouse of `StrictMode`, the useEffect is fired twice in dev mode. This simple solution prevents this from happening :)